pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -200,4 +205,8 @@ def auto_gain(self, setval):
         else:
             self.write("AUTOMATIC 0")
 
-    # TODO: shutdown function. Not sure what defaults should be, if any needed for it.
+    def shutdown(self):
+        """ Ensures the instrument in a safe state """
+        self.voltage = 0.
+        self.isShutdown = True

```

Comment:
This is the correct variable set in the `Instrument` class. However, it would be great to get the code in line with Python standards and use snake case -- preferring `is_shutdown`. I think that is worth doing in a future PR.